index.js

```diff
@@ -59,24 +62,33 @@ debug("Trying to read template '%s'", template);
 fs.readFile(template, function (err, templateContent) {
 	if (err) {
 		require("optimist").showHelp();
-		console.error("\nUnable to locate template file " + argv._[1]);
+		console.error("\nUnable to locate template file " + template);
 		process.exit(5);
 	} else {
 		getOptions(function (options) {
-			debug("Running git log in '%s' on branch '%s' with range '%s'", options.p, options.b, argv._[0]);
+			
+			debug("Running git log in '%s' on branch '%s' with range '%s'", options.p, options.b, range);
 			git.log({
 				branch : options.b,
-				range : argv._[0],
+				range : range,
 				title : new RegExp(options.t),
 				meaning : Array.isArray(options.m) ? options.m : [options.m],
 				cwd : options.p
 			}, function (commits) {
 				debug("Got %d commits", commits.length);
 				if (commits.length) {
 					debug("Rendering template");
+					
+					var rangeSince = range.split("..")[0];
```

Comment:
This won't work in all cases, the range can be any [git revision range](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection), I wouldn't do any parsing here, just pass the value down to the template